lib/frozen_record/test_helper.rb

```diff
@@ -27,7 +27,12 @@ def unload_fixture(model_class)
 
         old_base_path = @cache[model_class]
         model_class.base_path = old_base_path
-        model_class.load_records(force: true)
+
+        begin
+          model_class.load_records(force: true)
+        rescue
+          nil
```

Comment:
Rather than swallow the error it should be re-raised. 

Passing a wrong path shouldn't silently fail.

Reply:
That's the point: there isn't a "wrong" path.

Look, the fixture and frozen record only exists for the test, and can hence not be forcefully loaded on 'unload'

Reply:
k, something like:
during `load_fixture` if the model exists under the alternate path but not under the original/configured path => store it a bit differently (like *not* putting it into the @cache)?

Reply:
> if the model exists under the alternate path but not under the original/configured path => store it a bit differently (or just _not_ putting it into the @cache)?

Yes. e.g. store `false` or some other token value you can check in `unload`.

Reply:
voted for putting `nil` into the cache and only force-loading the original backend file when the cache has a non nil value. I think that expresses the idea quite well - there isn't a basepath to go back to, so it's nil. okay?